generic/tests/netperf.py

```diff
@@ -118,10 +119,67 @@ def mtu_set(mtu):
         if params.get("reboot_after_config", "yes") == "yes":
             vm.reboot(method="system_reset", serial=True)
 
-    try:
-        vm.wait_for_serial_login(timeout=login_timeout, restart_network=True).close()
-    except virt_vm.VMIPAddressMissingError:
-        pass
+    failover_exist = False
+    for i in params.get("nics").split():
+        nic_params = params.object_params(i)
+        if nic_params.get("failover_pair_id"):
+            failover_exist = True
+    if failover_exist:
```

Comment:
I am curious when there are multiple NICs, the value of `failover_exist` always be the latest one, what is this need?

Reply:
> I am curious when there are multiple NICs, the value of `failover_exist` always be the latest one, what is this need?

The default, we test with muliple nics, for only get "nic_params.get("failover_pair_id")" the failover_exist will be ture, for other scenarios, like bridge test, the failover_exist = False.

Reply:
So why get `failover_exist` repeatedly in a loop, the logic should be wrong.

Reply:
> So why get `failover_exist` repeatedly in a loop, the logic should be wrong.

ok, add break in the loop. and test it, it's passed. 

JOB LOG    : /root/avocado/job-results/job-2022-01-07T02.24-dcf4bcd/job.log
 (1/1) 82599.failover_test.1q.acceptance.repeat1.Host_RHEL.m8.u6.qcow2.virtio_blk.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.netperf.default.host_guest.q35: PASS (315.54 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2022-01-07T02.24-dcf4bcd/results.html